stable_baselines3/common/base_class.py

```diff
@@ -23,12 +25,30 @@
 from stable_baselines3.common.noise import ActionNoise
 
 
+def maybe_make_env(env: Union[GymEnv, str, None], monitor_wrapper: bool, verbose: int) -> Optional[GymEnv]:
```

Comment:
I don't know if it should be there, as a static method or in `utils.py`...
because this is only used by the `BaseAlgorithm` class.

Reply:
If only used by `BaseAlgorithm` then probably belongs somewhere in this file.

I'd be OK with module-level function or static method. I've got in the habit of avoiding `@staticmethod` after being [brainwashed by Google](https://google.github.io/styleguide/pyguide.html#2174-decision) but I don't think their argument is that strong.